### PR TITLE
django CMS 3.5 compatiblity

### PR DESCRIPTION
**Pull request by kynsi**
Modified apphook so it works with django CMS 3.5

### CODE REVIEW

File: aldryn_search/cms_apps.py [hunk 1]
```diff
@@ -6,10 +6,14 @@
 from .conf import settings
 
 
+@apphook_pool.register
 class AldrynSearchApphook(CMSApp):
-    name = _("aldryn search")
-    urls = ['aldryn_search.urls']
```
Comment (czpython) [inline aldryn_search/cms_apps.py]: this is still needed for < 3.2 compat

File: aldryn_search/cms_apps.py [hunk 2]
```diff
@@ -6,10 +6,14 @@
...
+    name = _("Aldryn Search")
+    app_name = "aldryn_search"
+    
+    def get_urls(self, page=None, language=None, **kwargs):
+        return ["aldryn_search.urls"]  
 
-
-if settings.ALDRYN_SEARCH_REGISTER_APPHOOK:
```
Comment (czpython) [inline aldryn_search/cms_apps.py]: this is still needed to make the apphook optional per project

File: setup.py [hunk 3]
```diff
@@ -5,9 +5,9 @@
 install_requires = [
     'lxml',
     'setuptools',
-    'Django>=1.4',
+    'Django>=1.8',
```
Comment (czpython) [inline setup.py]: I'd rather continue to support < 1.8 on the new patch release and then officially drop < 1.8 on a major release

File: setup.py [hunk 4]
```diff
@@ -5,9 +5,9 @@
...
     'django-appconf',
-    'django-cms>=3.0',
+    'django-cms>=3.5',
```
Comment (czpython) [inline setup.py]: we still support < 3.5

Comment (kynsi) [reply setup.py]: Is it somehow possible to make these changes optional? So only activate them if django-cms > 3.4?

**Comment by czpython** [comment]
Replaced by https://github.com/aldryn/aldryn-search/pull/84

**Pull request closed without merging**